Packages/vcs/Lib/configurator.py

```diff
@@ -915,3 +736,6 @@ def safe_get(obj, attr, sentinel=None):
         return getattr(obj, attr)
     except AttributeError:
         return sentinel
+
+def editable_type(display, key):
+    return display.g_type in ("marker", "text") or is_label(key)
```

Comment:
Most python implementations can handle when this doesn't happen, but all source code files should end with a newline character (see the symbol at the end of this line warning about this).